src/test/java/app/knock/api/resources/WorkflowResourceTestsIT.java

```diff
@@ -59,6 +59,28 @@ void testSuccessfulWorkflowTrigger() {
         assertNotNull(UUID.fromString(result.getWorkflowRunId()));
     }
 
+    @Test
+    void testIdempotentWorkflowTrigger() {
```

Comment:
Going to highlight that ci.yml does _not_ run the integration tests. Even so, I messed around with the tests to get them to run locally against my local switchboard, and can confirm it passed.